content/languages/clojure/clojure-test.md

```diff
@@ -2,3 +2,27 @@
 title: Clojure Test
 tags: [clojure]
 ---
+
+# Clojure Test
+
+## Basic Setup
+
+### Solution Code
+
+```clojure
+(ns clojure.adder)
+
+(defn add [a b]
+  (+ a b))
+```
+
+### Tests
+
+```clojure
+(ns clojure.adder-test
```

Comment:
We shouldn't use `clojure.` namespace for user's code.

Reply:
I have no idea about Clojure beyond copy-paste from existing Qualified docs. What namespace do you suggest? Something to the effect of `adder`/`challenge`/`solution`?

Reply:
Anything as long as it doesn't conflict with or belong to the standard library (`clojure`) and package namespaces.

I'd use `solution` and `solution-test` or `adder` and `adder-test`.